example/cumulus-tf/main.tf

```diff
@@ -112,6 +112,8 @@ module "cumulus" {
 
   token_secret = var.token_secret
 
+  es_request_concurrency = var.es_request_concurrency
```

Comment:
So we want to be able to set this specifically for the example deployments? Because the default value for the `cumulus` module is 10 already

Reply:
I was testing it so I used it

Reply:
I removed it